docs/book/src/clusterctl/configuration.md

```diff
@@ -48,3 +49,41 @@ AWS_B64ENCODED_CREDENTIALS: XXXXXXXX
 ```
 
 In case a variable is defined both in the config file and as an OS environment variable, the latter takes precedence.
+
+## Image overrides
+
+<aside class="note warning">
+
+<h1> Warning! </h1>
+
+Image override is an advanced feature and wrong configuration can easily lead to non-functional clusters.
+It's strongly recommended to test configurations on dev/test environments before using this functionality in production.
+
+</aside>
+
+When working in air-gapped environments, it's necessary to alter the manifests to be installed in order to pull
+images from a local/custom image repository instead of public ones (e.g. `gcr.io`, or `quay.io`).
+
+The `clusterctl` configuration file can be used to instruct `clusterctl` to override images automatically.
+
+This can be achieved by adding an `images` configuration entry as shown in the example:
+
+```yaml
+images:
+  all:
+    repository: myorg.io/local-repo
+```
+
+Please note that the image override feature allows for more fine-grained configuration, allowing to set image
+overrides for specific components, for example:
+
+```yaml
+images:
+  all:
+    repository: myorg.io/local-repo
+  cert-manager:
+    tag: v0.11.1 
```

Comment:
Will the image specific override take precedence over the `all` setting?

Reply:
@CecileRobertMichon 
The override works union, with the most specific override config taking precedence on the all config in case of conflicts.

So, in this example:
-  for cluster-api images, the "all" override apply to all the images in the cluster-api manifest; same for other providers
-  for cert-manager images, the "all" override U the "cert-manager" override apply to all the images in the cert-manager manifest